Formula/zenio.rb

```diff
@@ -0,0 +1,19 @@
+class Zenio < Formula
+  desc "CLI Zenio Tool"
+  homepage "https://dev.zenio.co"
+  url "https://dev.zenio.co/cli/zenio-0.8.1.tar"
+  sha256 "8838c70497d0d4e99365c32a7a197f90f2d3e67a8b1cab5e5a6ebac4f6d1bc92"
+
+  depends_on :java
+
+  def install
+    inreplace "zenio", "##PREFIX##", prefix
+    inreplace "zenio", "##VERSION##", "0.8.1-all"
+    prefix.install "zenio-0.8.1-all.jar"
+    bin.install "zenio"
+  end
+
+  test do
+    system "#{bin}/zenio", "version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
done